jsonsaver/saver2v2/save_files.go

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
+
+package saver2v2
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+
+	"github.com/spdx/tools-golang/spdx"
+)
+
+func renderfiles2_2(doc *spdx.Document2_2, buf *bytes.Buffer) error {
```

Comment:
super minor nit, I would capitalize the "F" in "Files" just to align with how it is for the other similar functions.

Fine to keep the "r" in "render" lowercase, since this won't be an exported function.